### DISCUSSION TITLE
Show longevity

### DISCUSSION THREAD

**Discussion by adamghill** [adamghill/unsuckjs.com#5] [Ideas]
Calculate the time from first commit to last commit?

Maybe there are other ways to determine that a library is well supported? Test coverage? Number of dependent projects?

**Comment by waldyrious** [comment]
I think a set of project liveness/robustness metrics would be useful, not just longevity. In particular, I would add number of (non-automated) open PRs, date of last merged PR, and perhaps the creation date of the oldest open PR.

It may be helpful to show the date-related metrics with relative dates (e.g. x days/weeks/months/years ago), in addition to the absolute dates (perhaps as a tooltip?). These relative dates can be calculated dynamically so that the data files don't need to be constantly updated.

These metrics could be grouped alongside the existing ones in the same vein (date of last commit, number of stars/watchers/forks, and number of open issues).

**Reply by adamghill** [reply]
I'd be interested in a PR for some of this functionality!

**Reply by waldyrious** [reply]
Nice, I'm glad you like those ideas. Just to be clear, I would love to work on this but I don't think I'll have the bandwidth to do so anytime soon... :see_no_evil:
